optaweb-vehicle-routing-backend/src/main/java/org/optaweb/vehiclerouting/service/vehicle/VehicleRepository.java

```diff
@@ -20,19 +20,26 @@
 import java.util.Optional;
 
 import org.optaweb.vehiclerouting.domain.Vehicle;
+import org.optaweb.vehiclerouting.domain.VehicleData;
 
 /**
  * Defines repository operations on vehicles.
  */
 public interface VehicleRepository {
 
     /**
-     * Create a vehicle with a unique ID.
-     * @param name vehicle name
+     * Create vehicle with a unique ID.
```

Comment:
comment nitpick: I believe removing the "a" article before "vehicle" is not grammatically correct.